src/main/java/com/hollingsworth/arsnouveau/setup/registry/APIRegistry.java

```diff
@@ -110,6 +111,18 @@ public static void setup() {
         registerSpell(EffectWall.INSTANCE);
         registerSpell(EffectAnimate.INSTANCE);
         registerSpell(EffectBurst.INSTANCE);
+
+
+        //not sure if these will be in the final PR
+        //on the one hand, I can see them being confusing to new players
+        //on the other hand, multiple addons are going to want to add them
+        //so there's going to be a ton of duplicate glyphs if these aren't in base ars
+        registerSpell(EffectContextEscape.INSTANCE);
```

Comment:
can these be moved to the dev registration only in the interim?

Reply:
done